### PR TITLE
Auto-fix Clippy pedantic warnings

### PR DESCRIPTION
**Pull request by nicokosi**
Via:

    cargo clippy --fix -- -Wclippy::pedantic

After:

    cargo clippy -- -Wclippy::pedantic 2>&1| grep "warning:" | wc -l
    285

Now:

    cargo clippy -- -Wclippy::pedantic 2>&1| grep "warning:" | wc -l
    112

PS: several fixes relates to my own code changes from #420. :innocent:

### CODE REVIEW

File: src/interface.rs [hunk 1]
```diff
@@ -866,7 +857,7 @@ impl<'a> Interface<'a> {
 
                 KeyEvent {
                     modifiers: KeyModifiers::CONTROL,
-                    code: Char('c') | Char('g') | Char('z') | Char('r'), // TODO add ZZ as shortcut
+                    code: Char('c' | 'g' | 'z' | 'r'),
```
Comment (cantino) [inline src/interface.rs:860]: Did you mean to remove this comment?

**Review by cantino — COMMENTED** [review]
I assume you tested and everything still seems to work?

File: src/interface.rs (hunk 1 above)
Comment (nicokosi) [reply src/interface.rs:860]: You're right, the comment should be kept.

**Comment by cantino** [comment]
Thanks!

**Pull request merged**